classes/Utils.php

```diff
@@ -10020,8 +10037,21 @@ public function error_message( $key = '401' ) {
 		$error_messages = apply_filters(
 			'tutor_default_error_messages',
 			array(
-				'401'   => __( 'You are not authorzied to perform this action', 'tutor' ),
-				'nonce' => __( 'Nonce not matched. Action failed!', 'tutor' ),
+				'401'                => __( 'You are not authorzied to perform this action', 'tutor' ),
+				'nonce'              => __( 'Nonce not matched. Action failed!', 'tutor' ),
+				'invalid_req'        => __( 'Invalid request', 'tutor' ),
+				'authentication'     => __( 'Authentication failed', 'tutor' ),
+				'authorization'      => __( 'Authorization required', 'tutor' ),
+				'not_found'          => __( 'Requested resource not found', 'tutor' ),
+				'server_error'       => __( 'Internal server error', 'tutor' ),
+				'timeout'            => __( 'Request timed out', 'tutor' ),
+				'forbidden'          => __( 'Access to this resource is forbidden', 'tutor' ),
+				'method_not_allowed' => __( 'HTTP method not allowed', 'tutor' ),
+				'too_many_requests'  => __( 'Too many requests', 'tutor' ),
+				'validation_error'   => __( 'Validation error', 'tutor' ),
```

Comment:
For validation error key, we can use generic message like `Invalid inputs` instead of Validation error.

Reply:
**validation_error** is there for this